robot/impl/resource_manager.go

```diff
@@ -646,7 +650,7 @@ func (manager *resourceManager) completeConfig(
 		case <-ctx.Done():
 			return
 		}
-	} // for-each resource name
```

Comment:
This method just needs to be rewritten -- but I do* prefer having this. It's a 100 line for-loop with up to 6 levels of nesting.

Reply:
I think the linter cut this off - will add back